.github/workflows/release_rust.yml

```diff
@@ -162,22 +162,79 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - base: debian
+          # stepflow-server images
+          - image: stepflow-server
+            base: debian
```

Comment:
Should we do alpine for musl (vs. glibc for debian?) - i guess these are different rust targets. I wonder which is most common, but I do know the alpine image will be smaller at least.

Reply:
Oh lol, we are doing both, which seems to be the concensus, nevermind.